vars/build.md

```diff
@@ -184,13 +196,15 @@ Details of how to resolve issues are covered within the (internal)
 confluence page. If the page is not accessible, see the documentation on
 [snyk.io](https://support.snyk.io/hc/en-us/articles/360003891038-Fix-your-vulnerabilities).
 
-Takes three parameters:
+Takes four parameters, the last one being optional:
 - `failOnIssues` - should the job fail when issues are detected. Default is
   `true`
 - `organisation` - against which the report should be recorded. Default is the
   env var `SNYK_ORG`
 - `severity` - of issues to be reported. Default is `medium`
+- `targetFile` - name of file to analyse
```

Comment:
Should we specify that this needs to be a `.sln` file or does it work with project files too?

Reply:
This is intended only as an example of passing a filename to the command, and isn't specific to .NET core -the same command can be used for Node.js, Python, Ruby and so on. 

I believe you can specify a `.proj` file, but not a `.csproj` file, though all the examples I've seen use a `.sln` file.